Casks/r/removebg.rb

```diff
@@ -7,9 +7,11 @@
   desc "Automatic bulk background removal"
   homepage "https://www.remove.bg/"
 
+  # The in-app updater returns a JSON file, but it is not a valid JSON file
+  # so we check the response with a regex using the :page_match strategy instead
   livecheck do
-    url "https://www.remove.bg/windows-mac-linux/download"
-    regex(%r{/removebg[._-]for[._-]mac[._-](\d+(?:\.\d+)+)\.dmg}i)
+    url "https://storage.googleapis.com/removebg-static/desktop/release/darwin/x64/RELEASES.json"
+    regex(/currentRelease"\s*:\s*"v?(\d+(?:\.\d+)+)[ "]/i)
```

Comment:
```suggestion
    regex(/"currentRelease"\s*:\s*"v?(\d+(?:\.\d+)+)"/i)
```

* We intend to match the `currentRelease` value, so using `"currentRelease"` (instead of omitting the leading double quote) makes sense to me.
* Unless upstream has published versions with a space in it, I think we can just keep this simple and use `"` instead of `[ "]`. The check will break if a version appears that doesn't use a strict `1.2.3` format (e.g., 1.2.3-4, etc.) but we would need to update the cask to handle a new version setup anyway and the failure would let us know that something's broken (instead of returning a partial version that's missing a suffix, which may not appear as a new version).

Reply:
Thanks @samford - hopefully they fix the json and we can simplify to just using the Json strategy at some point